pkg/pdutil/utils.go

```diff
@@ -15,13 +15,15 @@ package pdutil
 
 import (
 	"context"
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
 	"strconv"
 
 	cerror "github.com/pingcap/tiflow/pkg/errors"
 	pd "github.com/tikv/pd/client"
 )
 
-const sourceIDName = "source_id"
+const sourceIDName = "/global/config/source_id"
```

Comment:
It feels like a breaking change.

Reply:
- The previous implementation of PD's `global configuration` interface added "/global/config" internally.
	To Provide a way let the client can watch the configuration changes, config path is given externally. 	 https://github.com/tikv/pd/issues/5794
- For `CDC source_id feature` since tidb calls [the `Store Global` interface](https://github.com/pingcap/tidb/blob/master/domain/globalconfigsync/globalconfig.go#L26), source id calls pd `Load Config` interface
In order to be compatible with `source id` that previously had the `/global/config` prefix, it is necessary for CDC explicitly to add the `/global/config` here